script.js

```diff
@@ -12,7 +12,7 @@ $(window).on('load', function () {
                             <td>` + element.date + `</td>
                             <td>` + (element.price === "" ? '-' : element.price) + `</td>
```

Comment:
We can change "" to null, in which case we just leave the 'price' info blank.

Reply:
Yes, it's better to use `null` rather than an empty string `""` for the price, which is an integer:

While in the YAML file, we prefer to use `~` to denote an empty value:

```
key1:               # empty so "null", # is a comment!
key2: ~             # also "null", better
```